### PR TITLE
Contributors for Chapel 1.32

### PR DESCRIPTION
**Pull request by bradcray**
This is the list of people who contributed to Chapel 1.32—thanks everyone!

### CODE REVIEW

**Comment by bradcray** [comment]
@Guillaume-Helbecque , @Iainmon, and @Stinners : Thanks for your contributions to Chapel 1.32!  If you could check that I've spelled and alphabetized your names correctly, that would be great.

@chapel-lang/core-contributors : If you merged PRs from other contributors who I missed, please let me know.

**Comment by Guillaume-Helbecque** [comment]
@bradcray: Looks good, thanks! I'm just curious about the difference between "external" and "individual" contributor?

**Comment by bradcray** [comment]
@Guillaume-Helbecque : Whoops, good catch!  No difference was intended, I just apparently forgot which term we use when I added your entry.  Sorry about that!

**Comment by Stinners** [comment]
@bradcray, looks good!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The following people have contributed to Chapel releases:
 Contributors to the Chapel 1.31.0 release
```
Comment (riftEmber) [inline CONTRIBUTORS.md:6]: make sure to bump version before merge

Comment (bradcray) [reply CONTRIBUTORS.md]: Done, thanks!  (I'm thinking I should just take the mention of the version out given how often I forget this... but didn't do it yet).

**Pull request merged**